core/src/utils/platform.ts

```diff
@@ -33,14 +33,25 @@ const detectPlatforms = (win: Window) =>
 const isMobileWeb = (win: Window): boolean =>
   isMobile(win) && !isHybrid(win);
 
-const isIpad = (win: Window) =>
-  testUserAgent(win, /iPad/i);
+const isIpad = (win: Window) => {
+  // iOS 12 and below
+  if (testUserAgent(win, /iPad/i)) {
+    return true;
+  }
+
+  // iOS 13+
+  if (testUserAgent(win, /Macintosh/i) && isMobile(win)) {
```

Comment:
wondering if we can use some JS media query?

Reply:
Ok